Makefile

```diff
@@ -35,10 +35,10 @@ package: build
 		--url "https://github.com/uw-ictd/haulage" \
 		--deb-build-depends libpcap-dev \
 		--deb-compression gz \
-		--deb-systemd ./init/haulage.service \
```

Comment:
Why was this needed? Does it have to do with preinstall/postinstall ordering?

Reply:
Ah, I see the justification is in the PR. I'm not sure if restart after upgrade is incorrect or not in this case. If there are script conflicts then I'm fine with moving to the more manual install done in this commit.